TMS.Api/TMS.Api/Controllers/OrderController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using TMS.Api.Models;
+using TMS.Api.Models.DTOs;
+using TMS.Api.Repositories;
+
+namespace TMS.Api.Controllers
+{
+    [Route("api/[controller]/[action]")]
+    [ApiController]
+    public class OrderController : ControllerBase
+    {
+        private readonly IOrderRepository _orderRepository;
```

Comment:
la fel ca si la EventController, ar fi indicat sa se creeze un OrderService unde sa adauga toate logica de bussiness, in cazul de face in acel OrderService putem face referinta atat la OrderRepository cat si la TicketCathegoryRepository